Parser/tokenizer.c

```diff
@@ -856,6 +857,25 @@ tok_nextc(struct tok_state *tok)
                 if (translated == NULL)
                     return EOF;
                 newtok = translated;
+                if (tok->stdin_content == NULL) {
+                    tok->stdin_content = PyMem_Malloc(strlen(translated) + 1);
```

Comment:
Don't you need to add a a free operation to `PyTokenizer_Free`? for this new field?

Reply:
That's right. Done in https://github.com/python/cpython/pull/24140/commits/bdfc2a4df2cff9e2be8be13fd71f78e35f1283b1.